sections/05_treat.md

```diff
@@ -162,45 +162,193 @@ feature selection and construction.*
+alternative methods as thoroughly as in other domains.  DeepTox was the top
+performer on only 9 of 15 targets, and in some cases there was little separation
+between DeepTox and the second best method on those 9 tasks.  A reliance on AUC
+ROC `TODO: define here?` for the evaluation (see Discussion) further hinders the
+ability to declare Tox21 as an outright success for deep learning.
+
+In retrospect, the nuanced Tox21 performance may be more reflective of the
+practical challenges encountered in ligand-based chemical screening than the
+extreme enthusiasm generated by the Merck competition.  A study of 22 absorption
+distribution, metabolism, excretion, and toxicity (ADMET) tasks demonstrated
+that there are limitations to multitask transfer learning that are in part a
+consequence of the degree to which tasks are related [@tag:Kearnes2016_admet].
+Some of the ADMET datasets showed far superior performance in multitask models
+of only the 22 ADMET tasks relative to more expansive multitask networks that
+included over 500 less-similar tasks.  `TODO: also has a good discussion of
+information leakage in cross validation but that may be too detailed` In
+addition, training datasets encountered in practical applications may be tiny
+relative to what is available in public datasets and organized competitions.  A
+study of BACE-1 inhibitors included only 1547 compounds
+[@tag:Subramanian2016_bace1].  Machine learning models were able to train on
+this limited dataset, but overfitting was a challenge and the differences
+between random forests and a deep neural were negligible, especially in the
+classification setting.  Overfitting is still a problem in larger chemical
+screening datasets with tens or hundreds of thousands of compounds because the
+number of active compounds can be very small, on the order of 0.1% or 1% of all
+tested chemicals `TODO: verify those estimates`. This is consistent with the
+strong performance of low-parameter neural networks that emphasize
+compound-compound similarity, such as influence-relevance voter,
+[@tag:Swamidass2009_irv  @tag:Lusci2015_irv] instead of predicting compound
+activity directly from chemical features. `TODO: include recent DeepChem IRV
+benchmarks?`
+
+Much of the recent excitement in this domain has come from what could be
+considered a creative experimentation phase, in which deep learning has offered
+novel possibilities for feature representation and modeling of chemical
+compounds.  A molecular graph, where atoms are nodes and bonds are edges, is a
+natural way to represent a chemical structure.  Traditional machine learning
+approaches relied on preprocessing the graph into a feature vector, such as a
+fixed-width bit vector fingerprint [@tag:Rogers2010_fingerprints] in which each
+bit represents the presence of absence of a particular chemical substructure in
+the molecular graph. `TODO: this is a massive oversimplification because there
```

Comment:
Maybe mention that this is an oversimplification and give a reference for more detail? I think that if I was getting into the domain I'd want to know this, but as a non-domain reader I'd probably scan by it.